dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/world/model-info.xtxtuml

```diff
@@ -0,0 +1,151 @@
+ * {@link hu.elte.txtuml.api.stdlib.world.WorldObject WorldObject}s in the model which
+ * belong to the single {@link hu.elte.txtuml.api.stdlib.world.World World}
+ * instance and have unique {@code String} identifiers.
+ * Signals sent to these world objects are then handled by specific listeners. 
+ * <p>
+ * Note that as {@code World} and {@code WorldObject} are model classes, their instances are always
+ * created in and belong to a specific model execution. That is, the {@code World} is singleton
+ * and world object identifiers are unique only in the context of a specific model execution.
+ * Multiple parallel model executions are not related in any way.
+ * 
+ * <h1>How to use</h1>
+ * 
+ * <h2>Shared between the model and the outside world</h2>
+ * 
+ * There are only two things that need to be shared between the model and the outside world.
+ * <ul>
+ * <li>The strings that identify the outside entities and their the world objects that represent them.<br/>
+ * For this, static fields or methods are recommended to ensure thread safety.</li>
+ * <li>The signal types to be sent.<br/>
+ * These signal types have to be part of the model and <b>must extend
+ * {@link hu.elte.txtuml.api.stdlib.world.SignalToWorld SignalToWorld}</b>. Apart from simple Java
+ * visibility, nothing else is required for them to be used in the outside world as well.</li>
+ * </li>
+ * </ul>
+ * 
+ * <h2>Send signals from the model</h2>
+ * 
+ * Simple as it can be. Just:
+ * <ol>
+ * <li>Create and instance of the required signal (that extends {@code SignalToWorld}).</li>
+ * <li>Get the world object for the appropriate identifier. For this, simply call {@link 
+ * hu.elte.txtuml.api.stdlib.world.World#get World.get}.</li>
+ * <li>Call {@link hu.elte.txtuml.api.model.Action#send(Signal, ModelClass) Action.send} as always.</li>
+ * </ol>
+ * 
+ * <h2>Handle signals outside</h2>
+ * 
+ * <ol>
+ * <li>Subclass one of the subclasses of {@link 
+ * hu.elte.txtuml.api.stdlib.world.WorldObjectListener WorldObjectListener}.<br/>
+ * This is the step that has to be done with some care as different situations require different
+ * listeners to ensure thread safety. Take a look at the predefined versions or use
+ * {@link hu.elte.txtuml.api.stdlib.world.AbstractWorldObjectListener AbstractWorldObjectListener} if
+ * nothing other helps.</li>
+ * <li>In the listener, create signal handler methods that will take the desired signal as parameter.
+ * They also have to be {@code public} and have the
+ * {@link hu.elte.txtuml.api.stdlib.world.WorldObjectListener.SignalHandler SignalHandler} annotation.</li>
+ * <li>Call the {@link hu.elte.txtuml.api.stdlib.world.WorldObjectListener#register
+ * WorldObjectListener.register} to register for signals of the world object that has the specified 
+ * identifier and belongs to the given model executor.</li>
+ * <li>Wait for signals to arrive.</li>
+ * </ol>
+ * <h2>Send response to the model</h2>
```

Comment:
Won't fix. It does not have to be _a_ response, it can consist of any number of signals. Not to mention that I do not intend to speak about a certain response to a certain message here, but generally about responding to the model.